pynecone/event.py

```diff
@@ -272,6 +272,22 @@ def set_local_storage(key: str, value: str) -> EventSpec:
     )
 
 
+def set_clipboard(content: str) -> EventSpec:
+    """Set the text in content in the clipboard.
+
+    Args:
+        content (str): The text to add to clipoard
```

Comment:
Don't need to add type annotations in docstrings. Also small typo on "clipboard" and can we put a period at the end